src/NServiceBus.Core/obsoletes-v8.cs

```diff
@@ -1032,7 +1032,7 @@ public interface IPushMessages
             TreatAsErrorFromVersion = "8",
             RemoveInVersion = "9")]
         Task Init(Func<MessageContext, Task> onMessage,
-            Func<ErrorContext, Task<ErrorHandleResult>> onError,
+            Func<ErrorContext, Task<ReceiveResult>> onError,
```

Comment:
Does this indicate that we need to keep `ErrorHandleResult` in obsoletes for v8 (with an error)?

Reply:
yea, that needs to be kept as ErrorHandleResult